### ISSUE TITLE
Sidebar not sliding correctly

### ISSUE THREAD

**Issue by JFRN** [poole/lanyon#161]
Basically, what I want to do is to have the sidebar slide like the [Lanyon website](http://lanyon.getpoole.com/), but this is what I get. (this is has been happening since I installed vanilla Lanyon)
**Chrome**
![Chrome][chrome]
**Firefox**
![firefox][firefox]
Only the sidebar slides, and the site moves abruptly, unlike it is intended.
In Firefox, the sidebar toggle doesn't move. 

I'm really new to web development, so any help would be appreciated.
My site is [here](https://github.com/JFRN/jfrn.github.io).

[chrome]: http://i.imgur.com/ikVcKRL.gif
[firefox]: http://i.imgur.com/b2mjxNc.gif

**Comment by xHN35RQ** [comment]
How does it act when you visit this version of Lanyon? http://lanyon.getpoole.com/

**Comment by JFRN** [comment]
@xHN35RQ That effect is what I want to get.

**Comment by xHN35RQ** [comment]
Ah I see now. You're welcome to try my fork https://github.com/xHN35RQ/lanyon and see if there is any improvements. My fork is updated with various fixes from different Github authors.

**Comment by JFRN** [comment]
Thanks! Now the problem is that the toggle button doesn't change its color. But the slide effect has been fixed.

**Comment by xHN35RQ** [comment]
Hmm, something is messed up in your CSS - using a fresh clone of my fork it works just fine for me locally. But when I visit your site, it's not working. So compare the CSS for your site against https://github.com/xHN35RQ/lanyon and see if there are any differences

**Comment by JFRN** [comment]
Still the same, I am using the exact same CSS of your repository. 

I don't know where the issue could be :/.

**Comment by xHN35RQ** [comment]
Sorry I don't know what could be the matter. However, I know my fork works. You might consider restarting from scratch using my fork and re-doing any customizations on top of it.

**Comment by JFRN** [comment]
I'll try to fix it when I feel like it sometime, I'm still customizing things. But, thanks for the help!.

I will close this as it is probably only my case. (though I did use vanilla Lanyon)

**Comment by svaderia** [comment]
Hey @JFRN, I am facing the same issue as you described here, can you please tell me how did you fix this issue?  
Since user ghost has bee deleted and I'm not able to find his repo, It would be very helpful if you can guide me through this issue!

**Comment by JFRN** [comment]
https://xkcd.com/979/

@svaderia, sorry for being here too late. I didn't fix it completely, I don't remeber exactly what I did, though I tried using CSS from other Lanyon forks, navigating through the original CSS, and the slider issue disappeared, but then the button didn't change color, as I said two comments above. I'm sorry for not being so helpful there, but that's what I did. Check out @ghost fork and other sites that use lanyon, you might find a solution.

Good luck!
